build/commands/lib/pushL10n.js

```diff
@@ -11,14 +16,7 @@ const pushL10n = (options) => {
     options.with_translations ? '--with_translations' :
       options.with_missing_translations ? '--with_missing_translations' : ''
   if (options.extension) {
-    const extensionPath = options.extension_path
-    if (options.extension === 'greaselion') {
-      l10nUtil.getGreaselionScriptPaths(extensionPath).forEach((sourceStringPath) => {
-        util.run('python', ['script/push-l10n.py', '--source_string_path', sourceStringPath], cmdOptions)
-      })
-      return
-    }
-    console.error('Unknown extension: ', options.extension)
+    console.error('No extension is supported now')
```

Comment:
Is there any reason not to just drop this option entirely?

Reply:
Agree, let's just drop the whole thing.